spring-xsuaa/pom.xml

```diff
@@ -97,7 +97,22 @@
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
-
+		<dependency>
```

Comment:
I do not like the order... pls add this dependency before the "test" dependencies.